specs/core/0_beacon-chain.md

```diff
@@ -878,61 +857,39 @@ def get_seed(state: BeaconState, epoch: Epoch, domain_type: DomainType) -> Hash:
     return hash(domain_type + int_to_bytes(epoch, length=8) + mix)
 ```
 
-#### `get_committee_count`
+#### `get_committees_per_slot`
 
 ```python
-def get_committee_count(state: BeaconState, epoch: Epoch) -> uint64:
+def get_committees_per_slot(state: BeaconState, slot: Slot) -> uint64:
     """
-    Return the number of committees at ``epoch``.
+    Return the number of committees at ``slot``.
     """
-    committees_per_slot = max(1, min(
-        SHARD_COUNT // SLOTS_PER_EPOCH,
+    epoch = compute_epoch_of_slot(slot)
+    return max(1, min(
+        MAX_COMMITTEES_PER_SLOT,
         len(get_active_validator_indices(state, epoch)) // SLOTS_PER_EPOCH // TARGET_COMMITTEE_SIZE,
     ))
-    return committees_per_slot * SLOTS_PER_EPOCH
 ```
 
-#### `get_crosslink_committee`
+#### `get_beacon_committee`
 
 ```python
-def get_crosslink_committee(state: BeaconState, epoch: Epoch, shard: Shard) -> Sequence[ValidatorIndex]:
+def get_beacon_committee(state: BeaconState, slot: Slot, index: CommitteeIndex) -> Sequence[ValidatorIndex]:
     """
-    Return the crosslink committee at ``epoch`` for ``shard``.
+    Return the beacon committee at ``slot`` for ``index``.
     """
+    epoch = compute_epoch_of_slot(slot)
+    committees_per_slot = get_committees_per_slot(state, slot)
+    epoch_offset = index + (slot % SLOTS_PER_EPOCH) * committees_per_slot
```

Comment:
It is the number of committee in the epoch out of the total committees.

Say there are 4 slots in the epoch and 10 committees per slot. There are a total of 40 committees from the shuffling and we want to figure out which committee out of the 40 it is to get the correct shuffling.  So if "committee index" of 5 in the last slot in the epoch, it would be the `(10*3 + 5)` (35) of the 40

Reply:
Will work on the name or add a comment